airflow/providers/telegram/CHANGELOG.rst

```diff
@@ -24,6 +24,19 @@
 Changelog
 ---------
 
+4.0.0
+.....
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+* This release of provider only supports ``python-telegram-bot 20.0.0`` and above. All previous versions will
+  not work with ``4.0.0``.
```

Comment:
Preferably we should say what users require to do/look for. See other providers change log for reference. For example https://github.com/apache/airflow/blob/main/airflow/providers/amazon/CHANGELOG.rst#700

The information of `Updated Telegram Provider to ensure compatibility with >=20.0.0` will be shown as part of the commit message added tot he log. Here we are asking to provide some more context into what specifically users should look for. It's also OK to link to telegram github/docs if they provide information on how to migrate.

Reply:
I think more important part not bump to new major version of package, rather then fact that `TelegramHook.connection` public attribute now provide incompatible object, all other part seems like fine and compatible with previous one.

Reply:
I'm under the impression that if users upgrade this provider without core upgrade, for example, it breaks their existing code because python-telegram-bot < 20.0.0 will not work with the new telegram hooks code

Reply:
Not sure I understand you. If user install new version of provider by `pip install apache-airflow-providers-telegram==4.0.0` then it would also upgrade `python-telegram-bot` to an appropriate version